api/nakadi-event-bus-api.yaml

```diff
@@ -459,76 +407,16 @@ paths:
             future versions of this API.
 
           required: false
-          type: string
-          format: serialized json array of '#/definitions/Cursor'
-        - name: batch_limit
-          in: query
-          description: |
-            Maximum number of `Event`s in each chunk (and therefore per partition) of the stream.
-
-            * If 0 will buffer Events indefinitely and flush on reaching of `batch_flush_timeout`.
-          type: integer
-          format: int32
-          required: false
-          default: 1
-        - name: stream_limit
-          in: query
-          description: |
-            Maximum number of `Event`s in this stream (over all partitions being streamed in this
-            connection).
-
-            * If 0 or undefined, will stream batches indefinitely.
-
-            * Stream initialization will fail if `stream_limit` is lower than `batch_limit`.
-          type: integer
-          format: int32
-          required: false
-          default: 0
-        - name: batch_flush_timeout
-          in: query
-          description: |
-            Maximum time in seconds to wait for the flushing of each chunk (per partition).
-
-            * If the amount of buffered Events reaches `batch_limit` before this
-            `batch_flush_timeout` is reached, the messages are immediately flushed to the client and
-            batch flush timer is reset.
-
-            * If 0 or undefined, will assume 30 seconds.
-          type: number
-          format: int32
-          required: false
-          default: 30
-        - name: stream_timeout
-          in: query
-          description: |
-            Maximum time in seconds a stream will live before being interrupted.
-            If value is zero, streams indefinitely.
-
-            If this timeout is reached, any pending messages (in the sense of `stream_limit`) will
-            be flushed to the client.
-
-            Stream initialization will fail if `stream_timeout` is lower than `batch_flush_timeout`.
-          type: number
-          format: int32
-          required: false
-          default: 60
-        - name: stream_keep_alive_limit
-          in: query
-          description: |
-            Maximum number of keep-alive messages to get in a row before closing the connection.
-
-            If 0 or undefined will send keep alive messages indefinitely.
-          type: integer
-          format: int32
-          required: false
-          default: 0
-        - name: X-Flow-Id
-          in: header
-          description: |
-            The flow id of the request, which is written into the logs and passed to called
-            services. Helpful for operational troubleshooting and log analysis.
-          type: string
-          format: flow-id
+          type: array
```

Comment:
Have there been any changes to the params or is using refs insteaed just editorial?